src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -293,11 +293,10 @@ public void Track(ITelemetry telemetry)
             if (this.IsEnabled())
             {
                 this.Initialize(telemetry);
-                
-                this.WriteTelemetryToDebugOutput(telemetry);
```

Comment:
I remember we discussed that we would like to set an "IN" flag on a telemetry item before processing and then clear it if the item made it all the way to transmission. This way we know that the items with this flag remaining are the ones that are filtered or sampled out, so we can have additional prefix before them in the output window. 
This obviously only applies to the telemetry items with ikey present, when ikey is not present, all of them will appear in the output.